source/nodejs/adaptivecards-site/themes/adaptivecards/layout/_partial/header.ejs

```diff
@@ -442,7 +442,7 @@
         <div class="action-bar">
             <ul class="action-list">
                 <li>
-                    <a role="button" id="feedback-button" class="button" data-bi-name="feedback" tabindex="0" focusable="true">
```

Comment:
For my own learning, why don't we need to assign a role of "link" or something here? I assume that's a default, but where is it coming from?

Reply:
yeah -- some element types have an implied role (`a` is `link`, `button` is, well... `button), but you can use the `role` attribute to override it. usually you see explicit role declarations on things like a `div` that has a click handler :)

Reply:
Ah, got it. The class="button" threw me as I was wondering why a button defaulted to link role and I missed that we were in a link element here.

Reply:
I forgot to point out that for purposes of accessibility, if a click action causes a navigation to occur (or a new tab to appear), it should be marked as `link`, even if it's visually a button. The opposite is true in that something that looks like a link should me marked as `button` if it simply performs some action on the current page.